perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -1481,6 +1482,22 @@ public Attribute getAttribute(PerunSession sess, String key, String attributeNam
 		}
 	}
 
+	@Override
+	public Map<String,String> getEntitylessStringAttributeMapping(PerunSession sess, String attributeName) throws InternalErrorException, AttributeNotExistsException {
```

Comment:
Maybe we could retrieve ``Map<String,Object>`` instead, to support also other types of attributes.